demo/src/main/java/com/orange/ods/demo/ui/about/AboutHtmlFileScreen.kt

```diff
@@ -51,7 +54,17 @@ fun AboutHtmlFileScreen(
                             view?.loadUrl("javascript:(function(){ document.body.style.padding = '${verticalPadding}px ${horizontalPadding}px' })();");
                         }
                     }
-                    loadUrl("${FILE_PATH}${item.fileName}")
+
+                    val fileContent = resources.openRawResource(item.fileRes)
+                        .bufferedReader()
+                        .use(BufferedReader::readText)
+                    val html = when (item.fileFormat) {
```

Comment:
The name of the file is incoherent now because we manage Html and Markdown. I think we can rename it in `AboutFileScreen`